ecs-cli/modules/cli/local/options/options.go

```diff
@@ -54,3 +56,21 @@ func ValidateFlagPairs(c *cli.Context) error {
 	}
 	return nil
 }
+
+// ContainerSearchInfo show the task definition filepath or arn/family message on local down.
+func ContainerSearchInfo(c *cli.Context) {
```

Comment:
I wonder if we should move this function to `down_app.go` since other commands won't be using it, I'm not sure if it should be in the `options` package.  Looks good to me otherwise.